modules/ezcontent_node/modules/ezcontent_smart_article/src/Form/ConfigForm.php

```diff
@@ -125,17 +167,39 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#upload_location' => 'public://companies-data/',
       '#default_value' => $config->get('summary_generator_data_file'),
     ];
+    // Select plugin type for image captioning.
+    $form['image_captioning_service'] = [
+      '#title' => t('Image Captioning Service'),
+      '#type' => 'select',
+      '#description' => $this->t('Please choose image captioning service type.'),
+      '#options' => $image_captioning_options,
+      '#default_value' => $config->get('image_captioning_service'),
+    ];
     $form['image_captioning_api_url'] = [
       '#type' => 'textfield',
-      '#title' => $this->t('Image Captioning API URL'),
-      '#description' => $this->t('Provide the API URL.'),
+      '#title' => $this->t('Srijan Image Captioning API URL'),
```

Comment:
I believe the name needs to be changed here.
```suggestion
      '#title' => $this->t('EZContent Smart Image Captioning API URL'),
```